app/workers/billing_worker.rb

```diff
@@ -60,8 +60,8 @@ def self.with_billing_batch(provider_id, billing_date, &block)
   # @param [Integer] provider_id
   # @param [String] time
   def perform(buyer_id, provider_id, time)
-    Rails.logger.info("Billing worker invoked for buyer #{buyer_id} of provider #{provider_id} at #{time}")
-    billing_results = Finance::BillingService.call!(buyer_id, provider_account_id: provider_id, now: time, skip_notifications: true)
+    Rails.logger.info("[billing] provider #{provider_id} buyer #{buyer_id}: BillingWorker#perform invoked at #{time}")
+    billing_results = Finance::BillingService.call!(buyer_id, { provider_account_id: provider_id, now: time, skip_notifications: true })
```

Comment:
This is a "bonus"... in preparation for Ruby 2.7 / 3.x upgrade.

It's in a separate commit, but I can remove it if you think it's not OK to have it here.

Reply:
I always agree on fixing things now and not later.